### PR TITLE
Fix/widget template to render func

### PR DESCRIPTION
**Pull request by purepear**
- When using this package as dependency in other projects, there was runtime error because the "mutt-widget" component was not compiled to render function.
Vue/Webpack by default doesn't compile dependencies(./node_module/). And this was missed because it's inline component in .js file (not .vue file)

- Untrack `dist/*` files

I'm pretty sure this is backwards compatible but i want your feedback first :) then i will change the version in package.json

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -110,15 +110,19 @@ export default {
...
-            @callback="callback"
-            :is="getWidget()"
-            v-bind:field="field"
-            v-bind:fieldIndex="fieldIndex"
-            v-bind:readonly="readonly"
-            ></component>
-      `,
+      render(h) {
+        return h(this.getWidget(), {
+          tag: 'component',
```
Comment (alexrussell) [inline src/main.js]: Is this required if the render function uses the first param to determine _what_ to render?

Comment (purepear) [reply src/main.js]: @alexrussell I'm not sure what you mean? what is not required and what is the first param are you talking about?

**Comment by andypattenden** [comment]
I've not seen any runtime errors when using this package as a dependency, assuming it's because we have [`runtimeCompiler`](https://cli.vuejs.org/config/#runtimecompiler) set in the Vue config for apps.

Do you have an example of where you're seeing the error?

File: src/main.js (hunk 1 above)
Comment (purepear) [reply src/main.js]: @alexrussell I think we can remove `tag: 'component'` :)

**Comment by alexrussell** [comment]
Good point @andypattenden - this is being implemented into a new app, so it may not have some of the config options that have entered other codebases over the years.

Still, if it's possible for us to not have to include the runtime compiler, all the better, right? (Also considering the OS nature of this package.)

**Comment by andypattenden** [comment]
@purepear, can we get this rebased and pushed out? It might mean we can drop the `runtimeCompiler` option in the app configs and reduce the bundle sizes a bit.

**Comment by purepear** [comment]
@andypattenden  I'll close this PR and create a new one that's not from a fork and that doesn't remove the `dist/*` files

**Pull request closed without merging**